onnxruntime/python/tools/transformers/models/stable_diffusion/README.md

```diff
@@ -89,12 +108,31 @@ cd diffusers
 pip install -e .
 ```
 
+### Setup Environment (ROCm)
+
+It is recommended that the users should run the model with ROCm 5.4 or newer and Python 3.9. Note that Windows is not
+supported for ROCm at the moment.
+
+```
+conda create -n py39 python=3.9
+conda activate py39
+pip install -r requirements.txt
```

Comment:
rocm ep use requirements-rocm.txt, requirements-rocm.txt and requirements.txt may needs update according to diffusers version.